test/e2e/fleetautoscaler_test.go

```diff
@@ -771,7 +770,7 @@ func defaultAutoscalerWebhook(namespace string, fixedReplicasEnabled string) (*c
 			Ports: []corev1.ServicePort{{
 				Name:       "newport",
 				Port:       8000,
-				TargetPort: intstr.IntOrString{StrVal: "autoscaler"},
+				TargetPort: intstr.FromInt(8000),
```

Comment:
I was doing some test locally, it seems to be working by using `intstr.FromString("autoscaler")` (and keeping the name: "autoscaler" L750

intstr.FromString add the type with it: `IntOrString{Type: String, StrVal: val}`, I've got it working locally, not sure how it works on the background, if you can also test on your side, otherwise it's good to me to keep it like this 😄 
```